### PR TITLE
Replace static Swagger-ui distribution with unpkg version

### LINKED ISSUE #1076: swagger-ui DOM XSS
The swagger-ui which is used by docsy is outdated and prune to a DOM XSS vulnerability.

### LINKED ISSUE #1520: Docsy doesn't support rendering Swagger UI using OpenAPI 3.1.X
We would like to render Swagger UI  using OpenAPI version 3.1.0 (which has been available since [February 2021](https://www.openapis.org/blog/2021/02/18/openapi-specification-3-1-released)), but it is not supported.

### PR DESCRIPTION
**Pull request by theletterf**
This PR removes the static assets for swagger-ui and adds unpkg includes instead. I've also edited the docs accordingly.

- Closes #1520
- Closes #1076

### CODE REVIEW

**Comment by theletterf** [comment]
@chalin @LisaFC Can't currently test the change. Could you have a look?

**Comment by LisaFC** [comment]
Sure, happy to help, looks good from a quick glance but agree we'll need to check it works - is there a reason why you can't test it in your own project?

**Comment by theletterf** [comment]
Currently don't have a local Docsy project running and got confused by `docsy-example`. What's the recommended way of testing features?

**Comment by theletterf** [comment]
@LisaFC Even though it's slightly out of scope, I've fixed the color issue and added a `_swagger.scss` file which can come in handy for customizing Swagger UI's look and feel a bit (or for hiding elements by using the `display` directive). WDYT?

**Comment by LisaFC** [comment]
Oh that looks great! I see what you mean about the background contrast issue - @chalin what do you think?

**Comment by theletterf** [comment]
@LisaFC Fixed that in https://github.com/google/docsy/pull/1604/commits/5f31dd2859f1f7f0359daccbf6e1aaf135e3caeb

**Review by chalin — APPROVED** [review]
Great, thanks! LGTM

**Comment by chalin** [comment]
@LisaFC - can you give your stamp of approval too?

File: layouts/swagger/baseof.html [hunk 1]
```diff
@@ -22,8 +22,8 @@
           </aside>
           <main class="col-12 col-md-9 col-xl-8 ps-md-5" role="main">
             {{ if not .Site.Params.ui.breadcrumb_disable }}{{ partial "breadcrumb.html" . }}{{ end }}
-            <script src="{{ "js/swagger-ui-bundle.js" | relURL }}"></script>
-            <script src="{{ "js/swagger-ui-standalone-preset.js" | relURL }}"></script>
+            <script src="https://unpkg.com/swagger-ui-dist@5.1.0/swagger-ui-bundle.js" crossorigin></script>
```
Comment (deining) [inline layouts/swagger/baseof.html:25]: I would propose to add the SRI hash here (as we do with other scripts).
Same for other files pulled in from `unpkg.com`.

**Review by deining — CHANGES_REQUESTED** [review]

File: layouts/swagger/baseof.html (hunk 1 above)
Comment (chalin) [reply layouts/swagger/baseof.html:25]: That works for me.

Comment (theletterf) [reply layouts/swagger/baseof.html]: Edited to add the SRI hash. Thanks!

TIL that you can append `?meta` to any package in unpkg to retrieve package metadata, including SRI hashes: https://github.com/mjackson/unpkg/issues/48#issuecomment-323161338. That, or using srihash.org.

Comment (deining) [reply layouts/swagger/baseof.html]: > Edited to add the SRI hash. Thanks!
> 
> TIL that you can append `?meta` to any package in unpkg to retrieve package metadata, including SRI hashes: [mjackson/unpkg#48 (comment)](https://github.com/mjackson/unpkg/issues/48#issuecomment-323161338). That, or using srihash.org.

Thanks for the info. Up to now, I didn't know that either.

**Review by deining — APPROVED** [review]
LGTM now.
Can you rebase your PR to HEAD of main, please?

**Comment by theletterf** [comment]
@deining Done

**Comment by theletterf** [comment]
Can't merge on my own!

**Comment by chalin** [comment]
@LisaFC - still awaiting your approval here

**Review by LisaFC — APPROVED** [review]
Looks fantastic - sorry for the late review.

**Pull request merged**